store/src/main/java/com/nytimes/android/external/store/base/FetcherOld.java

```diff
@@ -0,0 +1,24 @@
+package com.nytimes.android.external.store.base;
+
+
+import com.nytimes.android.external.store.base.impl.BarCode;
+
+import org.jetbrains.annotations.NotNull;
+
+import rx.Observable;
+
+/**
+ * Interface for fetching new data for a Store
+ *
+ * @param <Raw> data type before parsing
+ */
+public interface FetcherOld<Raw> extends Fetcher<Raw, BarCode> {
```

Comment:
`FetcherOld` doesn't describe the interface properly imo.
Consider renaming to something like `BarcodeFetcher`, etc.